daemon/daemon.go

```diff
@@ -169,6 +171,10 @@ func (c *Command) canAccess(r *http.Request, user *auth.UserState) accessResult
 		return accessOK
 	}
 
+	if c.RootOnly {
```

Comment:
It seems from above that /v2/users falls under GET, but there is no `!c.RootOnly` check under the GET conditional above. This seems to not be needed because we aren't setting c.GuestOK and c.RootOnly or c.UserOK and c.RootOnly at the same time, but as a hardening measure perhaps do:
```
if r.Method == "GET" && !c.RootOnly {
...
}
```
In this manner we default closed if c.RootOnly is set for the GETs.

Reply:
I've added that, and also added a panic if `RootOnly` is set together with any `*OK` property